x-pack/test/fleet_api_integration/apis/epm/install_remove_assets.ts

```diff
@@ -498,6 +505,10 @@ const expectAssetsInstalled = ({
     expect(sortedRes).eql({
       installed_kibana_space_id: 'default',
       installed_kibana: [
+        {
+          id: 'cloud-security-posture-rule-template',
+          type: 'cloud_security_posture_rule_template',
```

Comment:
id and type are wrong.
```suggestion
          id: 'sample_cloud_security_posture_rule_template',
          type: 'cloud-security-posture-rule-template',
```